tests/grid/test_create_network.py

```diff
@@ -34,7 +34,7 @@
     drainage_area=hynp.arrays(
         dtype=hynp.floating_dtypes(),
         shape=hynp.array_shapes(),
-        elements=floats(min_value=0, width=16),
+        elements=floats(min_value=0, max_value=1000, width=16),
```

Comment:
What's the max_value needed for?

Reply:
This might not be the best way to do this but I couldn't figure out another way. I wanted to run this test using floats ranging from 0 to a large number, while also testing a range of floating point types. Since 1000 can be represented with a 16bit float, I used that as the max. What I really would like to use is the max value for each floating point type.